### PR TITLE
our vision page

### PR DESCRIPTION
**Pull request by raoufswe**
- Adds our vision page

### CODE REVIEW

File: apps/pie-microsite/src/content/pages/all-about-pie/our-vision.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- Our output is for everyone: combining accessibility with the best design practices
+- We design and build with consumers, end users, creatives and engineers in mind
+- Our output makes our products more intuitive and easy to use
+
+### Effective
+
+- Our designs and components boost the performance of our products
+- We test and iterate based on JET expertise, research and competitor analysis
+- We set the bar for all UI development across JET
+
```
Comment (jamieomaguire) [inline apps/pie-microsite/src/content/pages/all-about-pie/our-vision.md:42]: can this extra newline be removed? Not a huge problem but curious :)

**Review by jamieomaguire — APPROVED** [review]

File: apps/pie-microsite/src/content/pages/all-about-pie/our-vision.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+### Consistent
+
+- We work methodically to ensure our design system is clear and user-friendly
+- We maximise our flexibility by adopting a modular approach
+- We give regular, timely progress updates to JET’s product-focused teams
+
+### Inclusive
+
+- Our output is for everyone: combining accessibility with the best design practices
```
Comment (LTurns) [inline apps/pie-microsite/src/content/pages/all-about-pie/our-vision.md:33]: probably one for design actually but should these bullet points have full-stops?

**Review by LTurns — DISMISSED** [review]

**Review by jamieomaguire — APPROVED** [review]

File: apps/pie-microsite/src/content/pages/all-about-pie/our-vision.md (hunk 2 above)
Comment (raoufswe) [reply apps/pie-microsite/src/content/pages/all-about-pie/our-vision.md:33]: yep just share it with them. 💯

Comment (jordifornellsJET) [reply apps/pie-microsite/src/content/pages/all-about-pie/our-vision.md:33]: Just amended the designs to add full stops, thanks!

**Pull request merged**